assets/blocks/lesson-actions/lesson-actions-block/block.json

```diff
@@ -3,5 +3,13 @@
   "category": "sensei-lms",
   "supports": {
     "html": false
+  },
+  "attributes": {
+    "activeBlocks": {
+      "type": "object",
```

Comment:
Maybe an array is more intuitive here?

Reply:
Yep! I agree with you that an array would be more intuitive here. I started implementing as array, but considering we toggle only some of the blocks, using an object end up making more sense, and the code is cleaner.

But getting your point, I renamed it to `toggledBlocks`. WDYT? Does it make more sense now?

Reply:
Not sure to be honest. I still think that it's more intuitive to add/remove a block to an array of blocks for such functionality and an array seems more appropriate for a collection of something. But it might be just me 😄

Reply:
I get your point. But I believe if we use the blocks as an array here, the code would become clunky. We would need to add all the possible blocks to the attribute (even we don't wanna toggle most of them), and trust in this array as source of truth to list the blocks. See this [part](https://github.com/Automattic/sensei/pull/3887/files#diff-fc8a7b975ce7ffe6080e2288eee909c3062feae71a5bd6f3a1a8899f9cb87932R118-R120), for example.